modules/machine-config-overview.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * operators/operator-reference.adoc
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="machine-config-overview{context}"]
+= MachineConfig overview
+
+The Machine Config Operator (MCO) lets you apply changes to
+the operating systems running on {product-title} nodes.
```

Comment:
This sounds a little off: "lets you apply changes to the operating systems"
I like "manages updates to systemd, cri-o/kubelet, kernel, NetworkManager, etc. It also offers a new MachineConfig CRD that can write configuration files onto the host." from https://github.com/openshift/machine-config-operator#machine-config-operator